### PR TITLE
Metal rods sprite updates depending on the amount of rods in the stack

### PR DESCRIPTION
**Pull request by unid15**
Sprites for 1, 2, 3, 4, 5 and more rods (from tg)

### CODE REVIEW

File: code/game/objects/items/stacks/rods.dm [hunk 1]
```diff
@@ -16,10 +16,26 @@
 	w_type=RECYK_METAL
 	melt_temperature = MELTPOINT_STEEL
 
+/obj/item/stack/rods/New()
```
Comment (ComicIronic) [inline code/game/objects/items/stacks/rods.dm]: But why

File: code/game/objects/items/stacks/stack.dm [hunk 2]
```diff
@@ -245,7 +252,7 @@
 			to_transfer = 1
 		else
 			to_transfer = min(S.amount, max_amount-amount)
-		amount+=to_transfer
+		src.add(to_transfer)
```
Comment (ComicIronic) [inline code/game/objects/items/stacks/stack.dm]: This isn't necessary here, update_icon() is called below.

**Comment by N3X15** [comment]
Conflicts

**Comment by DeityLink** [comment]
I like it but honestly it's gonna make them harder to click on when there's only one.

**Pull request closed without merging**